exercises/Anne-Hyacinthe/short-response.md

```diff
@@ -0,0 +1,12 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    A parameter is a placeholder for an argument while an argument is the variable being assessed.
```

Comment:
Yes, in essence. But arguments _can_ be variables but can also be plain values. Here is a language improvement.
> A parameter is a placeholder for an argument while an argument is the ~~variable being assessed~~ **value that gets passed to the function**.